test/shallowbackup.js

```diff
@@ -139,4 +139,31 @@ describe('#unit Perform backup using shallow backup', function() {
         assert.ok(couch.isDone);
       });
   });
+
+  it('should perform a shallow backup with attachments option', async function() {
+    const couch = nock(dbUrl)
+      // batch 1
+      .post('/_all_docs', { limit: 3, include_docs: true, attachments: true })
+      .reply(200, JSON.parse(fs.readFileSync('./test/fixtures/animaldb_all_docs_1.json', 'utf8')))
+      // batch 2
+      .post('/_all_docs', { limit: 3, start_key: badgerKey, include_docs: true, attachments: true })
+      .reply(200, JSON.parse(fs.readFileSync('./test/fixtures/animaldb_all_docs_2.json', 'utf8')))
+      // batch 3
+      .post('/_all_docs', { limit: 3, start_key: kookaburraKey, include_docs: true, attachments: true })
+      .reply(200, JSON.parse(fs.readFileSync('./test/fixtures/animaldb_all_docs_3.json', 'utf8')))
+      // batch 4
+      .post('/_all_docs', { limit: 3, start_key: snipeKey, include_docs: true, attachments: true })
+      .reply(200, JSON.parse(fs.readFileSync('./test/fixtures/animaldb_all_docs_4.json', 'utf8')));
+    return shallowBackup({ bufferSize: 3, parallelism: 1, attachments: true })
+      .then((summary) => {
+        // Assert the promise total
+        assert.strictEqual(summary.total, 11);
+        // Assert the correct number of written events
+        assert.strictEqual(counter, 4);
+        // Assert correct batche increments
```

Comment:
```suggestion
        // Assert correct batch increments
```

Same typo in several test cases in this file.

Reply:
Fixed in 454cc0c6d526b2d06f530f8f1731cddb0b1f557c